packages/styles/variables.css

```diff
@@ -8,6 +8,8 @@
   /* color palette */
   --white: #ffffff;
   --dark-workspace-color: #53636e;
+  --stroke-dark: #5d676f;
+  --background-dark: #53636e;
```

Comment:
This adds in styles not currently covered, but found in https://preview.uxpin.com/68df515d0023cc4b79f25d43f7663e24e363b1b9#/pages/141465411/simulate/sitemap